spynl/tests/test_exception.py

```diff
@@ -35,6 +40,46 @@ def make_response(self):
 
     return app
 
+class ToBeMapped(Exception):
+    """
+    dummy exception that stands in for an exception from spynl.models
+    """
+    def __init__(self, extra):
+        self.extra = extra
+
+    def __str__(self):
+        return 'A message from spynl.models'
```

Comment:
Since this is a generic feature and this is an externa project it might be better not to mention spynl.models :)